tests/test-cases.js

```diff
@@ -2018,6 +2027,51 @@ describe('Unparse Tests', function() {
 
 
 var CUSTOM_TESTS = [
+	{
```

Comment:
Could you please add new tests at the end?

Reply:
Yup! Done.